library/nextcloud

```diff
@@ -3,92 +3,47 @@
 Maintainers: Nextcloud <docker@nextcloud.com> (@nextcloud)
 GitRepo: https://github.com/nextcloud/docker.git
 
-Tags: 15.0.13RC1-apache, 15.0.13-rc-apache, 15.0-rc-apache, 15-rc-apache, 15.0.13RC1, 15.0.13-rc, 15.0-rc, 15-rc
+Tags: 15.0.13-apache, 15.0-apache, 15-apache, 15.0.13, 15.0, 15
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 15.0-rc/apache
-
-Tags: 15.0.13RC1-fpm-alpine, 15.0.13-rc-fpm-alpine, 15.0-rc-fpm-alpine, 15-rc-fpm-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 15.0-rc/fpm-alpine
-
-Tags: 15.0.13RC1-fpm, 15.0.13-rc-fpm, 15.0-rc-fpm, 15-rc-fpm
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 15.0-rc/fpm
-
-Tags: 15.0.12-apache, 15.0-apache, 15-apache, 15.0.12, 15.0, 15
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: cee1980750dbbe1c84d321aa0169a7c033156f2c
+GitCommit: fe5b6beaddff62255f246395576942f989a3903b
 Directory: 15.0/apache
 
-Tags: 15.0.12-fpm-alpine, 15.0-fpm-alpine, 15-fpm-alpine
+Tags: 15.0.13-fpm-alpine, 15.0-fpm-alpine, 15-fpm-alpine
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: cee1980750dbbe1c84d321aa0169a7c033156f2c
+GitCommit: fe5b6beaddff62255f246395576942f989a3903b
 Directory: 15.0/fpm-alpine
 
-Tags: 15.0.12-fpm, 15.0-fpm, 15-fpm
+Tags: 15.0.13-fpm, 15.0-fpm, 15-fpm
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: cee1980750dbbe1c84d321aa0169a7c033156f2c
+GitCommit: fe5b6beaddff62255f246395576942f989a3903b
 Directory: 15.0/fpm
 
-Tags: 16.0.6RC1-apache, 16.0.6-rc-apache, 16.0-rc-apache, 16-rc-apache, 16.0.6RC1, 16.0.6-rc, 16.0-rc, 16-rc
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 16.0-rc/apache
-
-Tags: 16.0.6RC1-fpm-alpine, 16.0.6-rc-fpm-alpine, 16.0-rc-fpm-alpine, 16-rc-fpm-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 16.0-rc/fpm-alpine
-
-Tags: 16.0.6RC1-fpm, 16.0.6-rc-fpm, 16.0-rc-fpm, 16-rc-fpm
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ce0bb40cd97da98b6d0f89089b8b1d1a8d0bb17b
-Directory: 16.0-rc/fpm
-
-Tags: 16.0.5-apache, 16.0-apache, 16-apache, stable-apache, 16.0.5, 16.0, 16, stable
```

Comment:
Stable tags are removed but not added, is that correct?

Reply:
yes